scripts/user_label_image.py

```diff
@@ -0,0 +1,161 @@
+"""
+███████╗███╗   ██╗ █████╗ ██████╗  ██████╗ █████╗ ████████╗
```

Comment:
I'm pretty sure this is the most valuable thing I've added to this project

Reply:
Consider a career in graphic design